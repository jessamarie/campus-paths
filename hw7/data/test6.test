r
5
unk
r
unk
5
r
130
Troy Building
r
Troy Building
130
r
130
unk
r
unk
130
q