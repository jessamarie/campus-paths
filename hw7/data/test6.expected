First building id/name, followed by Enter: Second building id/name, followed by Enter: 
Unknown building: [unk]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [unk]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [130]
First building id/name, followed by Enter: Second building id/name, followed by Enter: 
Unknown building: [130]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [130]
Unknown building: [unk]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [unk]
Unknown building: [130]
